Docs are insufficient

As Go doesn't install modules by default, the documentation doesn't work as shown.

```
$ go get github.com/lyraproj/hiera
package github.com/lyraproj/hiera: no Go files in /Users/jrhett/go/src/github.com/lyraproj/hiera

$ go install github.com/lyraproj/hiera/lookup
../../go/src/github.com/lyraproj/hiera/internal/location.go:8:2: cannot find package "github.com/bmatcuk/doublestar" in any of:
	/usr/local/Cellar/go/1.12.6/libexec/src/github.com/bmatcuk/doublestar (from $GOROOT)
	/Users/jrhett/go/src/github.com/bmatcuk/doublestar (from $GOPATH)
```

I suspect many potential users are not aware enough of Go's module management to do the clone of the repo and install the modules, so at least mentioning it is probably a good idea. Or outright mention `GO111MODULE=on go get github.com/lyraproj/hiera` as the quick fix ;)

--- a/README.md
+++ b/README.md
@@ -8,6 +8,11 @@ This module is a "clean-room" Go implementation of the Hiera framework, suitable
 
 ## How to use
 
+Hiera is a go module and go modules must be enabled by setting the environment variable GO111MODULE is on before an
+attempt is made to install:
+
+    export GO111MODULE=on
+
 #### Install the module
 
 To install the module under $GOPATH/src:

